Map/RoomUI.gd

```diff
@@ -32,4 +32,4 @@ func get_center_point() -> Vector2:
 	
 func _set_player_position_based_on_room():
 	PlayerManager.player_position = room.room_position
-	SignalBus.clicked_next_room_on_map.emit()
+	SignalBus.clicked_next_room_on_map.emit(self)
```

Comment:
there is no need for it to emit self, since the signal itself doesn't emit anything if you look at the signal bus

Reply:
I've updated the signal to callback `_on_room_clicked()` in MapUI to help with setting light_level

Reply:
Did you update the connection to this signal to drop parameters ?

Reply:
Ah I didn't update the signal bus itself, the connect did connect to the function with the new parameter. I will update SignalBus